modules/mod_ginger_edit/templates/logon_error/message.passcode.tpl

```diff
@@ -0,0 +1,3 @@
+<p>
+    {_ The two-factor authentication passcode did not match. Please check your entry and try again. _}
+</p>
```

Comment:
Isn’t this template one already in Zotonic? I see some other duplicates as well. Is that just to remove the logon_reminder link?

Reply:
Yes, because of the link.  Some messages have also textual variations.

Reply:
I can’t reproduce this. As a result of `mod_translation:generate/1`? In which directory?

Reply:
I assume you refer to the other discussion?